### DISCUSSION TITLE
Plesae help me to create a Vercel Account

### DISCUSSION THREAD

**Discussion by SolidStar994** [vercel/community#3456] [Help] [answered]
### Summary

Hello. Everybody.
I hope you are doing well.
I've just tried to create a Vercel Account with my Github, but my account has been blocked instantly.
I don't know why you blocked my Account and I hope you remove the restriction from my account.

### Steps to Reproduce

![Screenshot 2023-08-01 034353](https://github.com/vercel/community/assets/127972304/d61cef48-91f7-47e5-abdd-be26d7b96e34)

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
